app/javascript/packs/one-time-code-input.js

```diff
@@ -1,6 +1,19 @@
 import OneTimeCodeInput from '@18f/identity-one-time-code-input';
 
-const input = document.querySelector('.one-time-code-input');
-if (input) {
-  new OneTimeCodeInput(/** @type {HTMLInputElement} */ (input)).bind();
+const fakeField = /** @type {HTMLInputElement?} */ (document.querySelector('.one-time-code-input'));
+
+if (fakeField) {
+  const el = `<input type="hidden" name="code" id="code" class="hidden-input">`;
+  fakeField.insertAdjacentHTML('afterend', el);
+  fakeField.removeAttribute('name');
+  fakeField.removeAttribute('id');
+  const hiddenInput = /** @type {HTMLInputElement?} */ (document.getElementById(`code`));
+  if (hiddenInput && fakeField) {
+    fakeField.addEventListener('input', () => {
+      hiddenInput.value = fakeField.value;
+    });
+  }
```

Comment:
The partial allows for the `name` to be customized, and `code` is the default. We should either remove support for that customization, or maybe just inherit the name from the `fakeField` element. `insertAdjacentHTML` seems fine here, but always give me pause as a common source of XSS issues, so maybe we can create it programmatically instead, especially since we need a reference to the element anyways.

Also, at least in local development, we prefill the value, which it looks like isn't working correctly with these changes. Clicking "Submit" results in a "That security code is invalid" error. Maybe we should set the initial value too.

```suggestion
  const hiddenInput = /** @type {HTMLInputElement?} */ (document.createElement('input'));
  hiddenInput.name = fakeField.name;
  hiddenInput.value = fakeFIeld.value;
  /** @type {HTMLElement} */ (fakeField.parent).insertBefore(hiddenInput, fakeField);
  fakeField.removeAttribute('name');
  fakeField.addEventListener('input', () => {
    hiddenInput.value = fakeField.value;
  });
```